lib/index.js

```diff
@@ -5,10 +5,7 @@ var Resource = require('./Resource'),
     Controllers = require('./Controllers'),
     Errors = require('./Errors'),
     inflection = require('inflection'),
-    _ = require('lodash');
-
-// make deep defaults the default
-_.defaults = require('merge-defaults');
+    mergeDefaults = require('merge-defaults');
```

Comment:
Lodash is shared because It is used by sequelize and epilogue. When `_.defaults` is replaced, this is anywhere.
You can checkout the branch before this commit and see the problem in tests.
Anyway, I think it is not good practice to override a function lodash.